dump/dump.go

```diff
@@ -248,7 +248,7 @@ func getProxyConfiguration(ctx context.Context, group *errgroup.Group,
 			consumerGroups, err = GetAllConsumerGroups(ctx, client, config.SelectorTags)
 		}
 		if err != nil {
-			if kong.IsNotFoundErr(err) {
+			if kong.IsNotFoundErr(err) || kong.IsForbiddenErr(err) {
```

Comment:
since we are just returning `nil` will they see a warning about this too?

Reply:
They will only see the status code in verbose mode. Do you think we should log a warning too?

Reply:
could be a good idea, but we can keep this as is for this PR.